contentpool_client.module

```diff
@@ -4,3 +4,82 @@
  * @file
  * Module file.
  */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function contentpool_client_form_remote_edit_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $remote = $form_state->getFormObject()->getEntity();
+
+  $form['actions']['submit']['#submit'] = array_merge(['contentpool_client_form_remote_edit_form_submit'], $form['actions']['submit']['#submit']);
+
+  $form['contentpool_client_settings'] = [
+    '#type' => 'details',
+    '#title' => t('Contentpool client settings'),
+    '#open' => TRUE,
+  ];
+
+  $form['contentpool_client_settings']['autoregister'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Is contentpool'),
+    '#description' => t('Whether the remote site is an instance of the contentpool distribution.'),
+    '#default_value' => $remote->getThirdPartySetting('contentpool_client', 'autoregister', 0),
+  ];
+
+  $form['contentpool_client_settings']['autopull'] = [
```

Comment:
this is not necessary any more and should be removed.
if is_contentpool it will be pulled if the drush command is used. if the autopull_interval is  set to not-never, cron will handle it also.